components/portal/main/partials/example-page.html

```diff
@@ -28,7 +28,7 @@
       margin: 4px 8px;
     }
   </style>
-  <h2>Example expanded mode widgets</h2>
+  <h2 style="margin-left:26px">Example expanded mode widgets</h2>
```

Comment:
Maybe do the same in line 58 for the "Example compact mode widgets" heading, for consistency? Maybe there should be a CSS class to abstract this style for re-use or so?

Reply:
Added to the second heading tag. I would argue for no CSS class because the presentation of the heading should be left up to whatever app is being developed with the framework (as such a class would *only* exist for the example page, for now).